### PR TITLE
docs(readme): add new rules as experimental

### PR DESCRIPTION
**Pull request by mohammedzamakhan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -287,47 +287,54 @@ Below you can find a recommended configuration which is based on the [Angular St
...
+| `no-life-cycle-call`                          |     Stable     |
+| `no-template-call-expression`                 |     Stable     |
+| `no-queries-parameter`                        |     Stable     |
+| `prefer-inline-decorator`                     |     Stable     |
+| `template-cyclomatic-complexity`              |     Stable     |
+| `pipe-prefix`                                 | _Experimental_ |
+| `no-conflicting-life-cycle-hooks`             | _Experimental_ |
+| `i18n`                                        | _Experimental_ |
+| `no-unused-css`                               | _Experimental_ |
+| `template-conditional-complexity`             | _Experimental_ |
```
Comment (wKoza) [inline README.md]: I think we can change the status to `Stable` for this rule;

Comment (mohammedzamakhan) [reply README.md]: @mgechev can you confirm this?

Comment (mgechev) [reply README.md]: 👍

Comment (rafaelss95) [reply README.md]: Hmm.. there's an open issue relating problems with this rule: #627.

**Comment by wKoza** [comment]
I have added the `blocked` label in order to don't merge before the good time

File: README.md (hunk 1 above)
Comment (rafaelss95) [reply README.md]: @wKoza :point_up_2:

**Comment by rafaelss95** [comment]
@wKoza @mgechev @mohammedzamakhan I moved the changes from PR #754 (`README.md` and` CONTRIBUTION.md`) to here. That way, there is no risk of messing documentation before we release a version with these new/modified rules.

**Pull request merged**